index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my portfolio</title>
+    <link rel="stylesheet" href="styling.css">
+    <link rel="stylesheet" media="(min-width: 768px)"  href="style.css">
+  </head>
+  <body>
+    <div class="container">
+        <header class="section1">
+          <nav class="navigation">
+            <div class="logo">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+            </div>
+            <section class="item-nav">
+              <a href="menu-items" class="menu-bar touch rough"><img src="mobile tag.png" alt="menu bar"></a>
+            </section>
+            <nav class="mail-button items">
+              <p class="mail touch rough"><img src="Union.png" alt="mail link"></p>
+              <p class="menu-items">
+                <section class="items touch rough">
+                  <p class="item1">portfolio</p>
+                  <p class="item2">about</p>
+                  <p class="item3">contact</p>
+                </section>
+              </p>
+            </nav>
+          </nav>
+        </header>
+      </section>
+      <main class="section2">
+        <article class="intro-text">
+          <p class="text1">Hey There.<br>I'm Chris.</p>
```

Comment:
- [ ] kindly consider adding this in h1 tag as its the main title of the page and it will be more clear to the readers